scripts/docker-test

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+#
+# Run all tests inside a Docker container
+#
+
+set -uC
+vimgodir=$(cd -P "$(dirname "$0")/.." > /dev/null && pwd)
+cd "$vimgodir"
+
+if [ $(docker container ls -a --filter name=vim-go-test | wc -l) -eq 1 ]; then
```

Comment:
This really shouldn't be guarded. Docker will cache the images and will only build layers as needed.

Reply:
But then we would have to destroy and re-create the container on every test run?

Reply:
Yes, but creating a container from an image is cheap. What's your concern?

Reply:
I don't know ... this is just how I've always done things.

Reply:
Just because a container with the expected names exists, doesn't mean the container shouldn't be built. Let docker do the work of figuring out whether the image layers are valid or not....